lib/simulator-xcode-6.js

```diff
@@ -612,6 +612,21 @@ class SimulatorXcode6 extends EventEmitter {
     return await settings.updateLocationSettings(this, bundleId, authorized);
   }
 
+  /**
+   * Enable/Disable reduce motion.
+   *
+   * @param {string} bundleId - The application ID to update.
+   * @param {boolean} reduceMotion - Whether or not to enable it.
+   */
+  async reduceMotion (reduceMotion = true) {
+    if (await this.isRunning()) {
```

Comment:
I still consider that restart should not be in this method. It should be the caller code, who is responsible for simulator reboot (and it has to be stated in the docstring, that the method has no effect unless the sim is rebooted if it is currently running)

Reply:
Yeah, I agree. We have (or used to have) code in appium-ios-driver / appium-xcuitest-driver that check the capabilities a user sends in to determine if any of them will require a sim reboot, and it handles the reboot after running the commands from this package. It should be up to the caller to do the reboot.